src/routes/+error.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+	import { page } from '$app/stores';
+</script>
+
+<h1>{$page.status}: {$page.error.message}</h1>
+
+{#if $page.status === 404}
+	<p>The page you are looking for does not exist.</p>
+{/if}
+{#if $page.status === 500}
+	<p>There was an internal server error. Please try again later.</p>
+{/if}
+
+<a href="/">Go to Home</a>
```

Comment:
Thanks for the PR.

I'm about to merge #47 which introduced some changes to the layout.
In order for this error page to look _visually not broken_ it will need to be updated to:

```suggestion
<script lang="ts">
	import { page } from '$app/stores';
	import EmptyMessage from '$lib/components/EmptyMessage.svelte';
</script>

{#if $page.status === 404}
	<EmptyMessage><strong>Error {$page.status}</strong> — The page you are looking for does not exist.</EmptyMessage>
{:else if $page.status !== 200}
	<EmptyMessage><strong>Error {$page.status}</strong> — There was an internal server error. Please try again later.</EmptyMessage>
{/if}
```